content/en/tracing/profiler/enabling/linux.md

```diff
@@ -27,53 +27,54 @@ The Datadog Profiler for Linux lets you collect profile data for applications ru
 
 ## Requirements
 
-- The Datadog Profiler requires Linux kernel v4.17+ on an `amd64` compatible processor. It does not support macOS, BSD, Windows, or other operating systems besides Linux v4.17 or later. 
-- The `/proc/sys/kernel/perf_event_paranoid` setting must *at most* 2, or `CAP_SYS_ADMIN` (or `CAP_PERFMON` on Linux v5.8 or later) must be granted to the profiling process or service container.
+The Datadog Profiler requires Linux kernel v4.17+ on an `amd64` compatible processor. It does not support macOS, BSD, Windows, or other operating systems besides Linux v4.17 or later. 
 
 ## Installation
 
 To begin profiling applications:
 
-1. Ensure you are running the Datadog agent version [7.20.2][1]+ or [6.20.2][2]+ and that you satisfy the profiler requirements.
+1. Ensure you are running the Datadog agent version [7.20.2][1]+ or [6.20.2][2]+ and that you satisfy the profiler requirements. The `/proc/sys/kernel/perf_event_paranoid` setting must *at most* 2, or `CAP_SYS_ADMIN` (or `CAP_PERFMON` on Linux v5.8 or later) must be granted to the profiling process or service container.  You can run `cat /proc/sys/kernel/perf_event_paranoid` to check the value of this parameter.  Modifying the parameter depends on your specific Linux distribution, but `echo 2 | sudo tee /proc/sys/kernel/perf_event_paranoid` will work for many popular distros.
```

Comment:
@sanchda could we make the perf paranoid setting step #1 and then move the agent install to step #2? Idea is you have the host ready to go before you start installing stuff on the machine!

Reply:
Good suggestion, good idea.  Thank you!